.github/prlabeller.yml

```diff
@@ -0,0 +1,21 @@
+# Add repo label to any change in any file
```

Comment:
This is just the example config from the GitHub action, right?
That does not suit for us, but that's not a problem.
I will take care of adapting the config accordingly.

Reply:
yah,
basically i was trying to give at least one label to any issue as 'repo'

Reply:
I see, but we don't use such a label
You can find an overview of all our labels [here](https://github.com/apache/streampipes/labels)
Feel free to improve the config, if you want to e

Reply:
sure,
i will try my level best